app/models/group.rb

```diff
@@ -0,0 +1,27 @@
+class Group < ApplicationRecord
+  validates :name,
+    :responsable_name,
+    :responsable_contact,
+    :from_date,
+    :to_date,
+    presence: true
+
+  validate :from_date_be_in_the_future
+  validate :to_date_after_from_date
+
+  private
+
+  def to_date_after_from_date
+    return if from_date.blank? || to_date.blank?
```

Comment:
Don't use conditionals in the end of a line. Use explicit if/else blocks

Reply:
Ok! Done in fixup in `e7def89`